src/libuv/libuv-windows.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>dnx-libuv-windows</id>
+        <version>0.0.1-pre</version>
+        <authors>Microsoft</authors>
```

Comment:
These look fine. I thought about it some more and we should include the `thirdpartynotices.txt` in here as well. Just copy the file and do the same think I did in https://github.com/aspnet/KestrelHttpServer/commit/c4bf10d6e9d77f75fd2a413c87a6186ac52d3c69.